pages/index.tsx

```diff
@@ -99,8 +98,6 @@ class Homepage extends React.Component<HomepageProps> {
 
                 <Giving i18n={i18n} t={t} tReady={tReady} />
 
-                <ChildrenVideo i18n={i18n} t={t} tReady={tReady} />
```

Comment:
cc Pastor asked to clean some of this up

Reply:
TODO: remove the component, it's styles (can be done separately)